packages/components/unpublished-development-types/global.d.ts

```diff
@@ -26,7 +26,6 @@ declare module '@glint/environment-ember-loose/registry' {
       EmbroiderUtilRegistry /*, other addon registries */ {
     // local entries
     LinkToExternal: typeof LinkTo;
-    // we have to use `ember-stargate` version `0.4.3` because version `0.5.0` causes a break in the tests for `ember-lts-3.28`
```

Comment:
This seems like an important reason to not make this change RN

Reply:
I installed this in nomad and all tests are still passing there as well https://github.com/hashicorp/nomad/pull/25750

I would propose we remove the comment completely, as I don't think it provides any relevant details at this time, but if there is a new comment you would like there, I would be happy to add it, just let me know!

Reply:
@MelSumner if the tests are passing, there's no need anymore of the comment (I was the one that added it, and at that time the tests were not passing).